test/integration/policy_multiline_templatization_test.go

```diff
@@ -128,17 +128,19 @@ var _ = Describe(
 				} {
 					By(fmt.Sprintf("Checking copied ConfigMap %s/%s on the %s cluster", configNamespace, name, cluster))
 					Eventually(
-						func() string {
+						func() (map[string]string, error) {
 							configMap, err := client.CoreV1().ConfigMaps(configNamespace).Get(
 								ctx, name+"-copy", metav1.GetOptions{},
 							)
-							if err != nil {
-								return ""
-							}
 
-							return configMap.Data["extraData"]
+							return configMap.Data, err
 						}, defaultTimeoutSeconds*2, 1,
-					).Should(Equal("exists!"))
+					).Should(
+						HaveKeyWithValue("extraData", "exists!"),
+						fmt.Sprintf("ConfigMap %s/%s on the %s cluster should have expected data",
+							configNamespace, name, cluster,
+						),
+					)
```

Comment:
I need to get in the habit of using descriptions like this, they're super helpful!

Reply:
Same! I feel like I'm slowly figuring out some better ways to test with Ginkgo, and this is definitely one of them.